TabloidMVC/Views/Shared/_Layout.cshtml

```diff
@@ -33,6 +33,9 @@
                             <li class="nav-item mx-0 mx-lg-1">
                                 <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" asp-area="" asp-controller="Account" asp-action="Logout">LOGOUT</a>
                             </li>
+                             <li class="nav-item mx-0 mx-lg-1">
+                                <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" asp-area="" asp-controller="Account" asp-action="Index">USER PROFILES</a>
```

Comment:
I think it would be a better UI to not show this nav bar item at all if the user is logged in as a non-admin-- what do you think? Feels a little misleading to show the non-admins this link and then be like SIKE NOTHING HERE FOR YOU. Can you add a conditional here so that it only shows up if the user is an admin?

Reply:
I made an attempt at this but had trouble accessing my repos in the Layout.cs.